src/kili/services/copy_project/__init__.py

```diff
@@ -1,14 +1,17 @@
 """
 Copy project implementation.
 """
+import itertools
+import logging
 from typing import Dict, Optional
 
-from kili import mutations, queries
-from kili.authentication import KiliAuth
+from tqdm import tqdm
```

Comment:
we import tqdm from kili/utils/tqdm in case one day we want to change the tqdm bar and apply it to the all app